tests/Makefile.common

```diff
@@ -537,8 +536,9 @@ $(PROJECT_FILE).decls-DynComp $(PROJECT_FILE).txt-cset \
   $(PROJECT_FILE).decls-DynComp-nojdk $(PROJECT_FILE).txt-cset-nojdk: $(DCOMP_PREMAIN) $(DCOMP_RT) \
 				$(SCRATCH_FOR_DAIKON)/.copy-timestamp $(INST_LIST_FILE)
 	@echo
-	@echo "### Run DynComp (${PROJECT_FILE})"
+	@echo "### Compile source and run DynComp (${PROJECT_FILE})"
 	@echo
+	@echo inst_Files: '$(INST_FILES)'
```

Comment:
Was this for debugging, or should it be retained?

Reply:
I just moved this line, along with the compilation of the test case, from the Chicory recipe to the DynComp one, but we could remove it if you wish.

Reply:
i commented out the line